.github/workflows/merge-me.yml

```diff
@@ -0,0 +1,29 @@
+name: 'dhis2: auto-merge (dependabot)'
+
+on:
+    check_suite:
+        types:
+            - completed
+    pull_request:
+        types:
+            - edited
+            - labeled
+            - opened
+            - ready_for_review
+            - reopened
+            - synchronize
+            - unlabeled
+            - unlocked
+
+jobs:
+    merge-me:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Merge me!
+              uses: ridedott/merge-me-action@master
+              with:
+                  # This must be used as GitHub Actions token does not support
+                  # pushing to protected branches.
+                  GITHUB_LOGIN: dependabot
+                  GITHUB_TOKEN: ${{secrets.GH_TOKEN}}
```

Comment:
We should probably use `secrets.DHIS2_BOT_GITHUB_TOKEN` (the org secret) here right?

Reply:
(to be fair the other actions in this repo still use the local GH_TOKEN secret, so this is consistent and should be eventually changed for all the actions)

Reply:
Yeah, `DHIS2_BOT_GITHUB_TOKEN` is the right one to use.